inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -93,6 +94,7 @@ Engine::~Engine() {
 }
 
 static void Transformation(ICNNNetwork::Ptr& clonedNetwork, const Config& conf) {
+    std::cout << "Now we will perform transformations in MKLDNN.\n";
```

Comment:
Please revert all your changes from the CPU plugin. I believe that this PR should not touch the CPU plugin.

Reply:
Fixed.